SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityCleaner.java

```diff
@@ -383,6 +383,7 @@ private boolean shouldRemoveLbState(SingularityTaskId taskId, SingularityLoadBal
       case UNKNOWN:
       case WAITING:
       case SUCCESS:
+      case INVALID_REQUEST_NOOP:
```

Comment:
In this case, we don't need to remove from Baragon b/c Baragon doesn't know about it, right?